test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperExecutionContextTest.cs

```diff
@@ -257,7 +259,7 @@ public void AddMinimizedHtmlAttribute_MaintainsHTMLAttributes()
         public void AddMinimizedHtmlAttribute_MaintainsHTMLAttributes_SomeMinimized()
         {
             // Arrange
-            var executionContext = new TagHelperExecutionContext("input", selfClosing: true);
+            var executionContext = new TagHelperExecutionContext("input", tagMode: TagMode.SelfClosing);
```

Comment:
Fine for this class.

Clarifying: Are you saying `TagHelperBlockRewriterTest` has sufficient coverage of `StartTagOnly`? Seems to be a great deal more `SelfClosing` despite the new `Rewrite_AllowsCompatibleTagStructures()` test.